<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/icons/icons/windowIcon.png</normaloff>:/icons/icons/windowIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true">border-color: rgb(218, 218, 218);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Менеджер проекта</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="btMasterOfImportData">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>30</horstretch>
              <verstretch>30</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Мастер импорта данных</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/icons/icons/dataAdd.png</normaloff>:/icons/icons/dataAdd.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btMasterAnalysisData">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>30</horstretch>
              <verstretch>30</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Мастер анализа данных</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/icons/icons/analysis.png</normaloff>:/icons/icons/analysis.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btMasterCreateGraph">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>30</horstretch>
              <verstretch>30</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Мастер построения графиков</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/icons/icons/graph.png</normaloff>:/icons/icons/graph.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btDeleteItemTree">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>30</horstretch>
              <verstretch>30</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/icons/icons/closeProject.png</normaloff>:/icons/icons/closeProject.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeSchema">
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="autoExpandDelay">
           <number>1</number>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Проект</string>
    </property>
    <addaction name="newProject"/>
    <addaction name="openProject"/>
    <addaction name="saveProject"/>
    <addaction name="saveAsProject"/>
    <addaction name="separator"/>
    <addaction name="closeProject"/>
    <addaction name="separator"/>
    <addaction name="closeProgram"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Данные</string>
    </property>
    <addaction name="importData"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Анализ</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolsSchema">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newProject"/>
   <addaction name="openProject"/>
   <addaction name="saveProject"/>
  </widget>
  <widget class="QToolBar" name="toolsData">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="importData"/>
  </widget>
  <action name="newProject">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/newProject.png</normaloff>:/icons/icons/newProject.png</iconset>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openProject">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/openProject.png</normaloff>:/icons/icons/openProject.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveProject">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/saveProject.png</normaloff>:/icons/icons/saveProject.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsProject">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/saveAsProject.png</normaloff>:/icons/icons/saveAsProject.png</iconset>
   </property>
   <property name="text">
    <string>Сохратить как</string>
   </property>
  </action>
  <action name="closeProject">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/closeProject.png</normaloff>:/icons/icons/closeProject.png</iconset>
   </property>
   <property name="text">
    <string>Закрыть проект</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="closeProgram">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/closeProgram.png</normaloff>:/icons/icons/closeProgram.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="importData">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/dataAdd.png</normaloff>:/icons/icons/dataAdd.png</iconset>
   </property>
   <property name="text">
    <string>Мастер импорта данных</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/export.png</normaloff>:/icons/icons/export.png</iconset>
   </property>
   <property name="text">
    <string>Экспортировать данные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Наивный Байесовский классификатор</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Метод опорных векторов</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Дерево классификации</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Метод k-ближайших соседей</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>newProject</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>new_project()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openProject</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_project()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveProject</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save_project()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAsProject</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save_project_as()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeProject</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close_project()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeProgram</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close_program()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importData</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>master_import_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>master_export_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btMasterAnalysisData</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>master_analysis_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btMasterOfImportData</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>master_import_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeSchema</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>is_select_tree_schema_item()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btDeleteItemTree</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>delete_item_tree()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btMasterCreateGraph</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>create_graphs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_3</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>naive_bayes_classifier()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_4</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cvm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_5</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>classification_tree()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_6</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>nearest_neighbors_classification()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>new_project()</slot>
  <slot>open_project()</slot>
  <slot>save_project()</slot>
  <slot>save_project_as()</slot>
  <slot>close_project()</slot>
  <slot>close_program()</slot>
  <slot>import_data()</slot>
  <slot>export_data()</slot>
  <slot>master_analysis_data()</slot>
  <slot>master_import_data()</slot>
  <slot>is_select_tree_schema_item()</slot>
  <slot>master_export_data()</slot>
  <slot>delete_item_tree()</slot>
  <slot>create_graphs()</slot>
  <slot>naive_bayes_classifier()</slot>
  <slot>cvm()</slot>
  <slot>classification_tree()</slot>
  <slot>nearest_neighbors_classification()</slot>
 </slots>
</ui>
